docs/output.md

```diff
@@ -25,6 +25,8 @@ results/
 |   ├── prodigal/
 |   ├── prokka/
 |   └── pyrodigal/
+├── function/
```

Comment:
What exactly do you mean here by function? I don't find that particularly descriptive and would'nt know what you mean by that necessarily.

Reply:
We now decided for 'protein_annotation'.